README.md

```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```

Comment:
@hollenberry I would go with @kytrinyx's suggestion to link to it. To explain for multiple OS and versions, it gets wordy pretty quickly. As an FYI the Django Girls tutorial is a good resource that helps a newcomer install Python, pip, etc. :smile: